src/js/package.json

```diff
@@ -61,7 +61,12 @@
     "./repodata.json": "./repodata.json"
   },
   "files": [
-    "pyodide*",
+    "pyodide.asm.js",
+    "pyodide.asm.wasm",
+    "pyodide_py.tar",
+    "pyodide.mjs",
+    "pyodide.js.map",
+    "pyodide.mjs.map",
```

Comment:
Could you confirm that these are only files included in the latest release (aside from those that were added by error as you mentioned)?

Reply:
```
console.html
distutils.tar
package.json
pyodide.asm.data
pyodide.asm.js
pyodide.asm.wasm
pyodide.d.ts
pyodide.js
pyodide.js.map
pyodide.mjs
pyodide.mjs.map
pyodide_py.tar
pyodide_tblib-1.7.1-py3-none-any.whl
README.md
repodata.json
```

These files are included in the 0.21.3 npm distribution. distutils.tar is removed after we changed the default behavior of loading fullstdlib (https://github.com/pyodide/pyodide/pull/2998). But I think we can think of including stdlibs in npm distribution (and also micropip and its dependents as suggested in https://github.com/pyodide/pyodide-webpack-plugin/pull/5#issuecomment-1258892445).